pkgs/by-name/ma/maltego/package.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, stdenv
+, fetchzip
+, jre
+, giflib
+, gawk
+, makeBinaryWrapper
+, icoutils
+, copyDesktopItems
+, makeDesktopItem
+}:
+
+stdenv.mkDerivation rec {
+  pname = "maltego";
+  version = "4.6.0";
+  src = fetchzip {
+    url = "https://downloads.maltego.com/maltego-v4/linux/Maltego.v${version}.linux.zip";
```

Comment:
are you sure this download link also works for macos since they publish another zip for that.

Reply:
Yes. I tested it in a Apple MacOS Sonoma (M1 processor)